package/yast2-bootloader.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Jan 27 10:59:26 UTC 2021 - José Iván López González <jlopez@suse.com>
+
+- Do not propose resume kernel parameter when the swap is smaller
+  than the RAM size (bsc#1180977).
```

Comment:
it makes sense to not propose it. Just description that it is not possible is a bit strong as not always whole ram is filled.